### DISCUSSION TITLE
Commandline vs VS IDE - Different file count when building

### DISCUSSION THREAD

**Discussion by KernelFrog** [YimMenu/YimMenu#3044] [Q&A] [answered]
Hi all, probably another dumb question on the build process, but I've noticed that building from the commandline.  e.g.
`cmake --build ./build --config RelwithDebInfo --target YimMenu --`

shows a different number of files compared to building from the VS2022 IDE, when building after a clean.

Commandline shows [787/787]  files whereas the IDE shows [825/825]. 

Any ideas what's going on?

**Comment by KernelFrog** [comment]
Bump?

**Comment by Yimura** [comment, marked as answer]
This is probably happening because the IDE is building for all targets whereas the command line is only compiling the YimMenu target.

Changing `--target YimMenu` to `--target all` will probably change the amount of files to match the one from the IDE. I haven't tested this however that's my best guess on why there's this difference.

**Reply by KernelFrog** [reply]
Thanks!  I think that was it. 
 `target clean`  followed by `target all` shows a file count of 822 (buildng the latest commit).
